docs/content_management/block_reference.md

```diff
@@ -292,7 +292,7 @@ On the **Properties** tab, set values in the following fields:
 
 - **Name** – Enter a name for the page block.
 
-On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+On the **Design** tab, in the **View** field, select the layout to be used to present the Sales representative's details and submit your changes.
```

Comment:
```suggestion
On the **Design** tab, in the **View** field, select the layout to be used to present the Sales representative's details page and submit your changes.
```

Reply:
But block is displayed as a section, we can go with the word: view.

Reply:
Makes sense, thanks!

Reply:
Added in https://github.com/ibexa/documentation-user/pull/302/commits/1c6d90424132982d8606bcff5c3bb9e88768813f